packages/dialtone-icons/src/icons/index.js

```diff
@@ -0,0 +1 @@
+export const icons = import.meta.glob('./*.vue', { import: 'default', eager: true });
```

Comment:
In theory if we turned off eager: true, this would dynamically load icons as they are used instead of loading them all up front? Obviously we'd have to test this, maybe a future upgrade though.

Reply:
Tried it, but Vue 3 was having issues to render the icons correctly, surely agree that it can be a future update.

Reply:
Yeah I think it might not be that simple, thanks for checking